assets/wire/components/filters/FiltersTab.jsx

```diff
@@ -135,20 +134,11 @@ class FiltersTab extends React.Component {
                         />)
                     ])}
                 </div>
-                {!isResetActive && !this.props.resultsFiltered ? null : ([                    
-                    <FilterButton
-                        key='search'
-                        label={gettext('Search')}
-                        onClick={this.search}
-                        className='filter-button--border'
-                        primary={true}
-                    />,
-                    <FilterButton
-                        key='reset'
-                        label={gettext('Clear filters')}
-                        onClick={this.reset}
-                        primary={false}
-                    />
+                {!isResetActive && !this.props.resultsFiltered ? null : ([
+                    <div className='tab-pane__footer' key='footer-buttons'>
+                        <button className='nh-button nh-button--primary' onClick={this.search}>{gettext('Search')}</button>
```

Comment:
This is a very long line now, which decreases readability. It would be good if we can have the attributes and tag body on different lines